homeassistant/components/ffmpeg_motion/binary_sensor.py

```diff
@@ -55,15 +56,16 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
 
 
 class FFmpegBinarySensor(FFmpegBase, BinarySensorEntity):
-    """A binary sensor which use FFmpeg for noise detection."""
+    """A binary sensor which use FFmpeg for detection."""
 
     def __init__(self, config):
-        """Init for the binary sensor noise detection."""
+        """Init for the ffmpeg binary sensor detection."""
         super().__init__(config.get(CONF_INITIAL_STATE))
 
         self._state = False
         self._config = config
-        self._name = config.get(CONF_NAME)
+        self._attr_name = config.get(CONF_NAME)
```

Comment:
Since `CONF_NAME` will always exist (with either a user-provided value or a default), you can simplify:

```suggestion
        self._attr_name = config[CONF_NAME]
```